platform/src/components/aws/bucket.ts

```diff
@@ -384,6 +441,20 @@ export interface BucketArgs {
    * @default `true`
    */
   cors?: Input<false | Prettify<BucketCorsArgs>>;
+  /**
+   * The lifecycle configuration for the bucket.
+   * @example
+   * This example will delete every object in the bucket after 30 days.
+   * ```js
+   * {
+   *  lifecycle: [{
+   *   enabled: true,
+   *   expiresIn: "30 days"
+   *  }]
+   * }
+   * ```
+   */
```

Comment:
```suggestion
   * Delete objects in the "/tmp" directory after 30 days.
   * ```js
   * {
   *   lifecycle: [
   *     {
   *       prefix: "/tmp",
   *       expiresIn: "30 days"
   *     }
   *   ]
   * }
```

formatted the array so it's clearer it allows multiple rules and added a prefix to show a common use case

Reply:
fixed